eng/release/Scripts/GetBarId.ps1

```diff
@@ -43,4 +43,4 @@ if ($TaskVariableName) {
         -Value $barId
 }
 
-Write-Ouptut $barId
\ No newline at end of file
+Write-Output $barId
```

Comment:
![GarfieldAnnoyedGIF](https://user-images.githubusercontent.com/19413848/152849430-2e4f19ba-1021-4617-bd7e-b5c34e2ba645.gif)

That's a good catch